scripts/superpmi.py

```diff
@@ -1520,6 +1554,166 @@ def print_platform_specific_environment_vars(coreclr_args, var, value):
     else:
         print("export {}={}".format(var, value))
 
+def download_index(coreclr_args):
+    """ Download the index.json for the collection.
+
+    Args:
+        coreclr_args (CoreclrArguments): parsed args
+
+    Notes:
+        The index.json file includes a dictionary of all of the different
+        collections that were done.
+    """
```

Comment:
Seems like it would be useful to include an example JSON file as documentation, or at least the schema.